src/test/java/com/cloudant/tests/UpdateHandlerTest.java

```diff
@@ -31,59 +31,57 @@
 
 public class UpdateHandlerTest {
 
-
-    private static Database db;
-    private CloudantClient account;
-
-
-    @Before
-    public  void setUp() {
-        account = CloudantClientHelper.getClient();
-        db = account.database("lightcouch-db-test", true);
-        db.syncDesignDocsWithDb();
-    }
-
-    @After
-    public void tearDown(){
-        account.shutdown();
-    }
-
-    @Test
-    public void updateHandler_queryString() {
-        final String oldValue = "foo";
-        final String newValue = "foo bar+plus=equals&ampersand";
-
-        Response response = db.save(new Foo(null, oldValue));
-
-        Params params = new Params()
-                    .addParam("field", "title")
-                    .addParam("value", newValue);
-
-        String output = db.invokeUpdateHandler("example/example_update", response.getId(), params);
-
-        // retrieve from db to verify
-        Foo foo = db.find(Foo.class, response.getId());
-
-        assertNotNull(output);
-        assertEquals(foo.getTitle(), newValue);
-    }
-
-    @Test
-    public void updateHandler_queryParams() {
-        final String oldValue = "foo";
-        final String newValue = "foo bar+plus=equals&ampersand";
-
-        Response response = db.save(new Foo(null, oldValue));
-
-        Params params = new Params()
-                    .addParam("field", "title")
-                    .addParam("value", newValue);
-        String output = db.invokeUpdateHandler("example/example_update", response.getId(), params);
-
-        // retrieve from db to verify
-        Foo foo = db.find(Foo.class, response.getId());
-
-        assertNotNull(output);
-        assertEquals(foo.getTitle(), newValue);
-    }
+	private static Database db;
+	private CloudantClient account;
+	
+
+	@Before
+	public  void setUp() {
+		account = CloudantClientHelper.getClient();
+		db = account.database("lightcouch-db-test", true);
+		db.syncDesignDocsWithDb();
+	}
+
+	@After
+	public void tearDown(){
+		account.deleteDB("lightcouch-db-test", "delete database");
+		account.shutdown();
+	}
+
+	@Test
+	public void updateHandler_queryString() {
+		final String oldValue = "foo";
+		final String newValue = "foo bar";
```

Comment:
It looks like in the update from master, the wrong version of this line setting `newValue` has been taken.